chemdash.py

```diff
@@ -1,3 +1,11 @@
+#!/usr/bin/env python3
+
+# NOTE: When this was written originally, using python3.7, "fork" was the default start method for multiprocessing.
+# The code currently fails when using the start method "spawn", which is now the default.
+# Explicitly set this until the issue is resolved.
+import multiprocessing
+multiprocessing.set_start_method("fork")
```

Comment:
As noted in the comments, I had to make this explicitly "fork" because the default method changed from "fork" to "spawn".  There are some quirks w/ using fork w/ flask, but it is the only option that runs as the code is now.  We can switch it when more things are updated internally.